playbooks/viya-mmsu/README.md

```diff
@@ -47,6 +47,23 @@ To restart all services gracefully, execute:
 ```
 ansible-playbook viya-ark/playbooks/viya-mmsu/viya-services-restart.yml
 ```
+To start the minimally required set of services prior to starting a Restore, execute:
+```
+ansible-playbook viya-ark/playbooks/viya-mmsu/viya-services-restore.yml
+```
+## Notes about the viya-services-start-restored.yml Playbook
```

Comment:
typo: `restored` instead of `restore`

also please add a blank line between the new section header and the line before it